canonical/ubuntu-pro-client
# tests: move landscape pending computer cleanup to after_all step

## Why is this needed?

This ensures pending computers are cleaned up even when ta landscape test fails partway through.

Fixes: #2676 

## Test Steps


```
tox -e behave-any -- features/landscape.feature
```
Check pending computers page on landscape: https://landscape.canonical.com/account/pro-client-qa/pending-computers
Ensure there are none

Edit a test in landscape.feature to fail (e.g. add an "When I run `false` with sudo" step somewhere.
Run the modified test.
Ensure there are no pending computers

## Checklist

 - [n/a] I have updated or added any unit tests accordingly
 - [n/a] Changes here need to be documented, and this was done in: 

## Does this PR require extra reviews?
